modules/day-time.js

```diff
@@ -0,0 +1,12 @@
+/* eslint-disable no-undef */
+
+let getTime;
+
+export default getTime = () => {
+  const dateAndTime = luxon.DateTime.now();
+  document.querySelector('.date').innerHTML = dateAndTime.toFormat('DD, HH:mm:ss');
+};
```

Comment:
- [ ] Kindly instead of disabling the linter you can import the `DateTime` object from the `luxon` file.

```javascript
import { DateTime } from './luxon.js';

let getTime;

export default getTime = () => {
  const dateAndTime = DateTime.now();
  document.querySelector('.date').innerHTML = dateAndTime.toFormat('DD, HH:mm:ss');
};

setInterval(() => {
  getTime();
}, 1000);
```